### ISSUE TITLE
Cluster Monitoring - Setting page does not sync the value from the monitoring app

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#19422]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- run Rancher v.2.2.2-rc2
- add a cluster
- enable cluster level monitoring
- go to project `system` -> workloads -> edit workload `exporter-node-cluster-monitoring` -> show advanced options -> Security & Host Config 
- change the values of `Memory Limit` and `CPU Limit`, then upgrade 
<img width="1334" alt="screenshot" src="https://user-images.githubusercontent.com/6218999/55656436-be957200-57ab-11e9-8d00-b0a2eff3d5d9.png">

- wait until the upgrade is finished, go to cluster monitoring setting page to check the variables
 

**Result:**
The values in the setting page are not updated to reflect the new values.
<img width="1301" alt="screenshot" src="https://user-images.githubusercontent.com/6218999/55656536-1af89180-57ac-11e9-978b-8605145b27db.png">

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
v2.2.2-rc2
- Installation option (single install/HA):
single install

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
- Kubernetes version (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.7", GitCommit:"6f482974b76db3f1e0f5d24605a9d1d38fad9a2b", GitTreeState:"clean", BuildDate:"2019-03-25T02:52:13Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.8-eks-7c34c0", GitCommit:"7c34c0d2f2d0f11f397d55a46945193a0e22d8f3", GitTreeState:"clean", BuildDate:"2019-03-01T22:49:39Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
17.6.2
```

**Comment by jiaqiluo** [comment]
Here is another scenario that is similar to the issue:
- run Rancher v2.2.1 
- add a cluster
- enable cluster monitoring
- upgrade Rancher to v2.2.2-rc2
- go to the cluster monitoring setting page
- notice that the monitoring page show two more variables `Node Exporter CPU Limit` and `Node Exporter Memory Limit` which actually does not take effect because they do not exist before upgrading. 
- If I clock on the `save` button` the monitoring app will upgrade to apply those two variables.

**Comment by alena1108** [comment]
For the original comment: @jiaqiluo the upgrade on the workload wouldn't change the values on the Cluster Monitoring configuration. It's driven the other way around. If you want to change the limits, it should be done on the ClusterMonitoring settings level. And then the workload will be started with this setting.

For the second comment, when the new changes would only apply when the save is clicked. @loganhz see if the UI can be smart and do not display "Node Exporter CPU Limit" default value if the monitoring has been previously enabled.

**Comment by loganhz** [comment]
For scenario one, there is nothing we can not keep them sync as we are using helm to deploy the app. Users still can upgrade workloads directly. So users need to do it in cluster monitoring settings.

For scenario two, the memory limit is 50 and cpu limit is 200 in 2.2.1.  I will put a fix. So in this case, UI will so 50 instead of the default value 200 in 2.2.2.

**Comment by jiaqiluo** [comment]
The bug fix is validated with Rancher: master 4fd2548
The steps are the same as https://github.com/rancher/rancher/issues/19422#issuecomment-480424082:

- run Rancher v2.2.1
- add a cluster
- enable cluster monitoring
- upgrade Rancher to master 4fd2548
- go to the cluster monitoring setting page

Result:
- notice that the monitoring page show two more variables `Node Exporter CPU Limit` and `Node Exporter Memory Limit` which are not showed in v2.2.1
- the value of `Node Exporter Memory Limit` is 50
<img width="1375" alt="screenshot" src="https://user-images.githubusercontent.com/6218999/56158578-441cdd00-5f77-11e9-9a13-1de4385043fa.png">
